pg_search/tests/pg_regress/expected/term_set_agg.out

```diff
@@ -0,0 +1,111 @@
+\i common/common_setup.sql
+CREATE EXTENSION IF NOT EXISTS pg_search;
+-- Disable parallel workers to avoid differences in plans
+SET max_parallel_workers_per_gather = 0;
+SET enable_indexscan to OFF;
+SET paradedb.enable_mixed_fast_field_exec = true;
+CREATE TABLE genus (
+  id BIGINT NOT NULL,
+  name TEXT NOT NULL
+);
+CREATE TABLE plants (
+  id BIGINT NOT NULL GENERATED ALWAYS AS IDENTITY,
+  genus_id BIGINT NOT NULL,
+  name TEXT NOT NULL
+);
+INSERT INTO genus (id, name) VALUES
+(0, 'oak'),
+(1, 'maple'),
+(2, 'pine'),
+(3, 'apple');
+INSERT INTO plants (genus_id, name) VALUES
+(0, 'English Oak'),
+(0, 'Holly Oak'),
+(0, 'White Oak'),
+(1, 'Sugar Maple'),
+(1, 'Red Maple'),
+(1, 'Norway Maple'),
+(2, 'Scots Pine'),
+(2, 'Ponderosa Pine'),
+(3, 'Domestic Apple'),
+(3, 'Siberian Crabapple');
+CREATE INDEX plants_idx ON plants
+USING bm25 (id, genus_id, name)
+WITH (key_field = id);
+CREATE INDEX genus_idx ON genus
+USING bm25 (id, name)
+WITH (key_field = id);
+--
+-- Test 1: Basic CTE query
+-- Find all plants belonging to the 'oak' genus.
+--
+-- NOTE: Using a term_set aggregate as the RHS of `@@@` is not supported in `0.18.x`.
+-- WITH genus_terms AS (
+--   SELECT pdb.term_set(id) as terms
+--   FROM genus
+--   WHERE genus.name @@@ 'oak'
+-- )
+-- SELECT plants.id, plants.name
+-- FROM plants, genus_terms
+-- WHERE plants.genus_id @@@ genus_terms.terms
+-- ORDER BY plants.id;
```

Comment:
FYI: I disabled this test rather than actually refactoring to support it in `0.18.x`.